src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -929,6 +929,14 @@ def load_arguments(self, _):
             help="Space-separated machine names to delete.",
         )
 
+    with self.argument_context("aks nodepool delete") as c:
+        c.argument(
+            "ignore_pod_disruption_budget",
+            options_list=["--ignore-pod-disruption-budget"],
```

Comment:
If the only option form is `--ignore-pod-disruption-budget`, you could ignore this line. In cli-extensions/aks-preview, it has another shorthand option name `-i`, do you want to keep it?

Reply:
Got it. I intentionally remove that to make sure CX fully understand what they're doing.

Reply:
Need to add ignore-pdb as ignore-pod-disruption-budget exceed the maximum length 22.